blob/drivertest/drivertest.go

```diff
@@ -581,65 +638,62 @@ func testListDelimiters(t *testing.T, newHarness HarnessMaker) {
 				listResult{Key: keyPrefix + "/f.txt"},
```

Comment:
Note to reviewer: `Hide Whitespace Changes` is your friend for this section of code, I just uncommented some tests.